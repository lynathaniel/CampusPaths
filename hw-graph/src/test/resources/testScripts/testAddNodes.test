# Tests for functionality of addNode method.

# Create a graph
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

# Add nodes to graph
AddNode graph1 n1

AddNode graph2 n1
AddNode graph2 n2

# Print the added nodes
ListNodes graph1
ListNodes graph2
ListNodes graph3
