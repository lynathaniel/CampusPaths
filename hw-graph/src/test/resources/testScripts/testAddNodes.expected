# Tests for functionality of addNode method.

# Create a graph
created graph graph1
created graph graph2
created graph graph3

# Add nodes to graph
added node n1 to graph1

added node n1 to graph2
added node n2 to graph2

# Print the added nodes
graph1 contains: n1
graph2 contains: n1 n2
graph3 contains: